modules/custom/dkan_common/src/Plugin/DataModifierBase.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Drupal\dkan_common\Plugin;
+
+use Drupal\Component\Plugin\PluginBase;
+
+/**
+ * Defines an abstract base class for Data modifier plugins.
+ */
+abstract class DataModifierBase extends PluginBase implements DataModifierInterface {
+
+  /**
+   * List of schemas to potentially modify. Others will be skipped.
+   *
+   * @var array
+   */
+  private $schemasToModify = [];
```

Comment:
This variable is set to private in an abstract class. I believe no child of this class will be able to modify it. We should make it protected or create a public setter and getter.

Reply:
https://github.com/GetDKAN/dkan2/pull/292